code/modules/mob/living/simple_animal/heretic_monsters.dm

```diff
@@ -0,0 +1,436 @@
+/mob/living/simple_animal/hostile/heretic_summon
+	name = "Eldritch Demon"
+	real_name = "Eldritch Demon"
+	desc = "A horror from beyond this realm."
+	icon = 'icons/mob/eldritch_mobs.dmi'
+	gender = NEUTER
+	mob_biotypes = NONE
+	attack_sound = 'sound/weapons/punch1.ogg'
+	response_help = "thinks better of touching"
+	response_disarm = "flails at"
+	response_harm = "reaps"
+	speak_emote = list("screams")
+	speak_chance = 1
+	speed = 0
+	stop_automated_movement = TRUE
+	AIStatus = AI_OFF
+	see_in_dark = 7
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = INFINITY
+	healable = FALSE
+	movement_type = GROUND
+	pressure_resistance = 100
+	del_on_death = TRUE
+	deathmessage = "implodes into itself."
+	faction = list(FACTION_HERETIC)
+	simple_mob_flags = SILENCE_RANGED_MESSAGE
+	/// Innate spells that are added when a beast is created.
+	var/list/spells_to_add
+
+/mob/living/simple_animal/hostile/heretic_summon/Initialize(mapload)
+	. = ..()
+	add_spells()
+
+/**
+ * Add_spells
+ *
+ * Goes through spells_to_add and adds each spell to the mind.
+ */
+/mob/living/simple_animal/hostile/heretic_summon/proc/add_spells()
+	for(var/spell in spells_to_add)
+		AddSpell(new spell())
+
+/mob/living/simple_animal/hostile/heretic_summon/raw_prophet
+	name = "Raw Prophet"
+	real_name = "Raw Prophet"
+	desc = "An abomination stitched together from a few severed arms and one lost eye."
+	icon_state = "raw_prophet"
+	icon_living = "raw_prophet"
+	status_flags = CANPUSH
+	melee_damage = 10
+	maxHealth = 50
+	health = 50
+	sight = SEE_MOBS|SEE_OBJS|SEE_TURFS
+	spells_to_add = list(
+		/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash/long,
+		/obj/effect/proc_holder/spell/pointed/manse_link,
+		/obj/effect/proc_holder/spell/targeted/telepathy/eldritch,
+		/obj/effect/proc_holder/spell/targeted/blind/eldritch,
+	)
+
+	/// A assoc list of [mob/living ref] to [datum/action ref] - all the mobs linked to our mansus network.
+	var/list/mob/living/linked_mobs = list()
+
+/mob/living/simple_animal/hostile/heretic_summon/raw_prophet/Initialize(mapload)
+	. = ..()
+	link_mob(src)
```

Comment:
confused because this seems to be the only use of link_mob, how would someone else get linked to this simplemob?

Reply:
It gets a spell to link other people to its 'hivemind chat'
![image](https://user-images.githubusercontent.com/110184118/225702355-69b249d2-1f47-4296-a0a5-6b411b1e8daf.png)